### DISCUSSION TITLE
cant install spotify

### DISCUSSION THREAD

**Discussion by naiyanrahman** [prasanthrangan/hyprdots#915] [Q&A]
I tried to install spotify but can't install it, I also tried to make from makepkg from this git https://github.com/kpcyrd/spotify-launcher still the same  
![image](https://github.com/prasanthrangan/hyprdots/assets/77504180/5a2c18fe-a50e-4f90-b380-a633fcaab2c6)

**Comment by kRHYME7** [comment]
``` yay -S spotify    ```

Please report it in your distribution if certain package won't compile/build/install. 

Hyprdots repo concerns are the .config files, they call it dotfiles.  You can still ask something in the Discussion panel though.  https://github.com/prasanthrangan/hyprdots/discussions

**Comment by kRHYME7** [comment]
Please reopen an issue relating to dotfiles, Cheers 🥂
